src/main/java/io/confluent/connect/jdbc/sink/dialect/HanaDialect.java

```diff
@@ -50,6 +54,8 @@ protected String getSqlType(String schemaName, Map<String, String> parameters, S
           return "DATE";
         case Timestamp.LOGICAL_NAME:
           return "TIMESTAMP";
+        default:
+          log.debug("unhandled schemaName={}", schemaName);
```

Comment:
Is the fix for this refactoring the interface such that each type would be individually dispatched and the generic type would handle the dispatch step + default case? If so, file a follow up to make that cleaner (as long as it doesn't affect compatibility).